ember/app/ui/cases/index/controller.js

```diff
@@ -130,12 +130,17 @@ export default class CasesIndexController extends TableController {
 
   @action
   selectCase(value) {
-    if (this.selectedCases.includes(value.id)) {
-      this.selectedCases = this.selectedCases.filter((id) => id !== value.id);
+    const deselection = this.selectedCases.filter((c) => c.id !== value.id);
+    if (deselection.length < this.selectedCases.length) {
+      this.selectedCases = deselection;
       return;
     }
```

Comment:
Why not?

```suggestion
    if (this.selectedCaseIds.includes(value.id)) {
      this.selectedCases = this.selectedCases.filter((c) => c.id !== value.id);
      return;
    }
```

Reply:
Would also work, but might as well already filter it when iterating over it.

Reply:
:rocket: 3 2 1...